.github/workflows/publish-a-package.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish a Package from Tag
+
+on:
+  push:
+    tags:
+      - 'opentelemetry-+==1.+'
```

Comment:
`+` should be a valid glob value for tags according to [GitHub's docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags)

Reply:
> [`+`: Matches one or more of the preceding character.](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#:~:text=character-,%2B%3A%20Matches%20one,of%20the%20preceding%20character,-Matches)

Wouldn't that only match the `-` and `.` respectively?

Reply:
Thank you! I wrongly assumed that it would work like the other globs but after reading the [Filter Pattern Cheat Sheet](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) I found out it does exactly what you say which isn't what I want.

I went with something simple and just used `*` instead, the script should be more informative as to why it fails if it does. That's probably preferred over the workflow just not triggering at all because it doesn't match.